UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,11 @@
 # Upgrade Guide
 
+## From `0.3.10` to `0.4.0`
+
+### Unity 2020.1
+
+In order to use the GDK for Unity `0.4.0`, you will need to upgrade your project to Unity 2020.1. We test against 2020.1.2f1, but any version _should_ work.
```

Comment:
```suggestion
In order to use the GDK for Unity `0.4.0`, you will need to upgrade your project to Unity 2020.1. We test against 2020.1.2f1, but any `2020.1.x` version _should_ work.
```

for ultra-clarity in case anyone misreads it 😅